### PR TITLE
Login Page

### PR DESCRIPTION
**Pull request by iamharnad**
Added login, registration toggle and fav icons

1. Added Navigation
2. Added Favicons
3. Added Login and Registration Form
4. Added Javascript inorder to toggle between registration and Login

image

### CODE REVIEW

**Review by Mahi24-prog — APPROVED** [review]
Looks good to me

File: account.html [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+            </ul>
+          </div>
+        </div>
+        <hr />
+        <p class="copyright">copyright 2020 amazon.com</p>
+      </div>
+    </div>
+
+    <script>
+      var MenuItems = document.getElementById('MenuItems');
```
Comment (Prabhav-Sharma) [inline account.html]: Adding a separate js file for code would be better

Comment (iamharnad) [reply account.html]: sure, will add

File: JS/form-toggle.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+var LoginForm = document.getElementById('LoginForm');
+var RegForm = document.getElementById('RegForm');
+var Indicator = document.getElementById('Indicator');
```
Comment (fahd23) [inline JS/form-toggle.js:3]: Why are you using var? Use const or let

File: JS/menu-toggle.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+MenuItems.style.maxHeight = '0px';
+
+function menutoggle() {
+  if (MenuItems.style.maxHeight == '0px') {
+    MenuItems.style.maxHeight = '200px';
+  } else {
+    MenuItems.style.maxHeight = '0px';
+  }
+}
```
Comment (fahd23) [inline JS/menu-toggle.js:11]: Why are you repeating the same code which is already in menu.js?

File: JS/cart.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+MenuItems.style.maxHeight = '0px';
+
+function menutoggle() {
+  if (MenuItems.style.maxHeight == '0px') {
+    MenuItems.style.maxHeight = '200px';
+  } else {
+    MenuItems.style.maxHeight = '0px';
+  }
+}
```
Comment (fahd23) [inline JS/cart.js:11]: I can see the same code in many files,can you tell me why

**Review by fahd23 — CHANGES_REQUESTED** [review]

File: JS/form-toggle.js (hunk 2 above)
Comment (iamharnad) [reply JS/form-toggle.js:3]: Nothing in particular, i will use new es6 concepts as you suggested

File: JS/cart.js (hunk 4 above)
Comment (iamharnad) [reply JS/cart.js:11]: Nothing special. Since I'm writing the code n pushing them to seperate branches individually

**Review by Mahi24-prog — APPROVED** [review]

**Review by iamharnad — COMMENTED** [review]
modified style.css as there are merge conflicts

**Review by ishapatell1 — APPROVED** [review]

**Review by ishapatell1 — APPROVED** [review]

**Comment by iamharnad** [comment]
resolved the merge conflicts

**Pull request closed without merging**